pkg/virt-controller/watch/volume-migration/OWNERS

```diff
@@ -2,5 +2,7 @@ reviewers:
   - sig-storage-reviewers
 approvers:
   - sig-storage-approvers
+options:
+  no_parent_owners: true
```

Comment:
Generally we don't want to use this option because we want the root approvers to be able to approve these...

Reply:
Right! :man_facepalming: I was not thinking that I would break the whole chain